packages/data-stores/src/site/types.ts

```diff
@@ -145,6 +145,30 @@ export interface SiteDetails {
 	isSecondaryNetworkSite?: boolean;
 }
 
+export enum SiteCapabilities {
```

Comment:
I see that Antonio mentioned the currently existing capabilities here: https://github.com/Automattic/dotcom-forge/issues/5088#issuecomment-1882949910 and it looks like these are the ones we are using here.

I am wondering if we should add the [missing capabilities](https://wordpress.org/documentation/article/roles-and-capabilities/) e.g. `install_themes` to be consistent in the capabilities filter or at least some of those that are missing and that we tried to use in the PR that caused the bug.

Reply:
I'm not exactly sure how we populate those user capabilities. The only reference of capabilities I've found is in this file.
I will open a new ticket to investigate more in-depth what's happening, as it may need changes to other places as well.
Thanks for sharing the capabilities docs.

Reply:
So far, I see that we are setting the capabilities in Jetpack: https://github.com/Automattic/jetpack/blob/98a8e97bbcf3306586f0a66b812268e540d470e7/projects/plugins/jetpack/sal/class.json-api-site-base.php#L902

But I have not looked yet further where exactly they are coming from.

Reply:
Yes, I also found that yesterday while I was investigating this issue, and I had the same question.
This is why I decided to move like that and investigate it at a later time ( in case my changes fix the issue for now, which it seems they did )

Reply:
For future reference, I created a new ticket https://github.com/Automattic/dotcom-forge/issues/5127

Reply:
Sounds like a plan - I agree that fixing the user-facing issue is the priority for now as it seems to be making some of the commands completely unusable.